Wikipedia/Code/Session.swift

```diff
@@ -74,7 +74,16 @@ public class Session: NSObject {
         }
     }
     
-    public var xWMFUUID: String? = nil // event logging uuid, set if enabled, nil if disabled
+    // event logging uuid, set if enabled, nil if disabled
+    private var xWMFUUID: String? {
+        get {
+            let userDefaults = UserDefaults.standard
+            if userDefaults.wmf_sendUsageReports {
+                return userDefaults.wmf_appInstallId
+            }
+            return nil
```

Comment:
Hmm, I guess as updated this will always be nil is sendUsageReports is false. Not sure if it's worth keeping the check in anyway or if it should be taken out.

Reply:
@mdholloway I'm good with keeping the check in just in case.